community/psmisc/APKBUILD

```diff
@@ -6,10 +6,13 @@ pkgdesc="A set of some small useful utilities that use the proc filesystem"
 url="http://psmisc.sourceforge.net/"
 arch="all"
 license="BSD GPL"
-depends=""
-makedepends="ncurses-dev"
+depends="libintl gettext"
+makedepends="gettext-dev ncurses-dev"
```

Comment:
depends isn't necessary, and see other comment instead of depending on gettext-dev